spec/presenters/archived_tag_presenter_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+RSpec.describe ArchivedTagPresenter do
+  
+  let(:parent)   { FactoryGirl.create :topic,
+                                      slug: 'parent',
+                                      title: 'Parent topic',
+                                      description: 'Description of parent topic.'
+                 }
+
+  let(:child)    { FactoryGirl.create :topic,
+                                      slug: 'child-1',
+                                      title: 'Child topic',
+                                      description: 'Description of child topic.',
+                                      parent: parent
+                 }
+
+  
+  let(:expected_child_content) {
+    {
+      content_id:     child.content_id,
+      base_path:      "/topic/parent/child-1",
+      format:         "redirect",
+      publishing_app: "collections-publisher",
+      update_type:    "major",
+      redirects: [
+        {
+          path:         "/topic/parent/child-1",
+          destination:  "/topic/parent",
+          type:         "exact"
+        },
+        {
+          path:         "/topic/parent/child-1/latest",
+          destination:  "/topic/parent",
+          type:         "exact"
+        },
+        {
+          path:         "/topic/parent/child-1/email-signup",
+          destination:  "/topic/parent",
+          type:         "exact"
+        },
+      ]
+    }
+  }
+
+ 
+
+
+  describe 'render_for_publishing_api' do
```

Comment:
Use `#render_for_publishing_api` for instance methods,`.render_for_publishing_api` for class methods.